index.html

```diff
@@ -376,113 +396,203 @@ <h4>Characters</h4>
           <p class="issue" data-number="15"></p>
-          <p>It MAY have a <dfn>Text Language</dfn> property. If not present, the <a>Text</a> is assumed to be in the Script <a>Primary Language</a>.</p>
-          <p class="note">In <a>Original Language Dialogue List</a>, <a>Events</a> may be in different languages.</p>
-        </section>
-
-        <section>
-          <h4>Contextual Text</h4>
-          <p>Each <a>Event</a> MAY have a <dfn>Contextual Text</dfn> property, which is a representation of the <a>Text</a> in a different language. It may be used to assist in the processing of the <a>Text</a>.
-          <p class="note">For example, when a <a>Pre-Recording Dub Script</a> needs to be adjusted, having this additional text in the original language(s) could be useful.</p>
-          <p class="issue" data-number="16"></p>
-          <p class="issue" data-number="21"></p>
-          <p>It MAY have a <dfn>Contextual Text Styles</dfn> property, which provides styles associated with the <a>Contextual Text</a> of this <a>Event</a>.</p>
-          <p class="issue" data-number="15"></p>
-          <p>It MAY have a <dfn>Contextual Text Language</dfn> property, representing the language of the <a>Contextual Text</a> content.</p>
-        </section>
-
-        <section>
-          <h4>Event Description</h4>
-          <p>Each <a>Event</a> MAY have an <dfn>Event Description</dfn> property, as a human-readable description of the <a>Event</a>. </p>
-          <p class="issue" data-number="30"></p>
-          <p class="note">It does not have to be unique for this <a>Event</a>. It may be used to identify in a human-readable way one or more <a>Events</a> that need to be processed together, e.g. in a batch recording.</p>
-          <p class="issue" data-number="20"></p>
+          <p>A <a>Text</a> object is represented with a <code>p</code> element with the following constraints:
+            <ul>
+              <li>The text content of the <code>p</code> is the <a>Text</a> of the <a>Event</a>. It MAY also use TTML markup to represent styles.
+                <div class="note">The text content of the paragraph may contain TTML elements such as <code>span</code> or TTML attributes such as <code>tts:ruby</code> used to alter the layout or styling of each paragraph</div>
+              </li>
+              <li>The <code>p</code> element MUST have an <code>xml:lang</code> attribute corresponding to the language of the <a>Text</a> object.</li>
+          <pre class="example">
+&lt;div xml:id=&quot;event_0130df43-5443-4206-b1d9-d331e42b5a86&quot;
+     begin=&quot;9663f&quot; end=&quot;9682f&quot; 
+     style=&quot;style_bb89e9c6-4453-4402-a376-5a496cfb9839&quot;
+     ttm:agent=&quot;character_bb89e9c6-4453-4402-a376-5a496cfb9839&quot;&gt;
+  &lt;p xml:lang=&quot;pt-BR&quot; &gt;Voc&#xea; vai ter.&lt;/p&gt;
+  &lt;p xml:lang=&quot;fr&quot; ttm:role=&quot;x-context&quot;&gt;Bah, il arrive.&lt;/p&gt;
+&lt;/div&gt;
+          </pre>
```

Comment:
Missing `</ul>`?

Is it not allowed to associate a Text with a Character, for Events with more than one Character?

Reply:
We need to clarify that unless the text is the same for all characters, each event should have only one character.